Exiled.API/Features/Scp330.cs

```diff
@@ -0,0 +1,147 @@
+// -----------------------------------------------------------------------
+// <copyright file="Scp330.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    using System.Collections.Generic;
+
+    using InventorySystem;
+    using InventorySystem.Items.Usables.Scp330;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// A Wrapper class for Scp-330.
+    /// </summary>
+    public class Scp330
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Scp330"/> class.
+        /// </summary>
+        /// <param name="scp330Bag"><inheritdoc cref="Base"/></param>
+        public Scp330(Scp330Bag scp330Bag)
+        {
+            Base = scp330Bag;
+            Room = scp330Bag.GetComponentInParent<Room>();
+        }
+
+        /// <summary>
+        /// Gets the base-game <see cref="Scp330Bag"/>.
+        /// </summary>
+        public Scp330Bag Base { get; }
+
+        /// <summary>
+        /// Gets the transform from the bag.
+        /// </summary>
+        public Transform Transform => Base.transform;
```

Comment:
Since you're getting the Transform, you should add the Position/Rotation as well.